### PR TITLE
Added more logging to IMU failure

### PR DESCRIPTION
**Pull request by ABenoit01**
## Description of Problem
Added more logging for gyro values in IMU failure.
Moved logging so it prints regardless of IMU result.
closes https://github.com/Harlem-Launch-Alliance/Catalyst-2/issues/39

## Testing
compiled code with no errors or warnings

### CODE REVIEW

File: main/gy521_imu.h [hunk 1]
```diff
@@ -70,14 +70,25 @@ void setupIMU(){
...
-    Serial1.println(sample.accel.x);
-    delay(1000);
-    Serial1.print("Y Acceleration: ");
-    Serial1.println(sample.accel.y);
-    delay(1000);
-    Serial1.print("Z Acceleration: ");
-    Serial1.println(sample.accel.z);
-    delay(1000);
-  }
+    }
```
Comment (JonathanSafer) [inline main/gy521_imu.h:73]: Could you shift this bracket to the left by one tab so it's lined up with the start of the word `else`?

**Review by JonathanSafer — CHANGES_REQUESTED** [review]
Last change and then you're all set

File: main/gy521_imu.h (hunk 1 above)
Comment (ABenoit01) [reply main/gy521_imu.h]: Done!

**Review by JonathanSafer — APPROVED** [review]
Nice work!

**Pull request merged**